tests/integrations/document/test_to_json.py

```diff
@@ -2,12 +2,12 @@
 import torch
 
 from docarray.document import BaseDocument
-from docarray.typing import AnyUrl, Tensor, TorchTensor
+from docarray.typing import AnyUrl, NdArray, TorchTensor
 
 
 def test_to_json():
     class Mmdoc(BaseDocument):
-        img: Tensor
+        img: NdArray
         url: AnyUrl
         txt: str
```

Comment:
can we add a Tensor field here ? To see if this Union does not break the to json